sdk/core/azure-core/azure/core/_pipeline_client_async.py

```diff
@@ -194,30 +193,13 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
         return AsyncPipeline(transport, policies)
 
     async def _make_pipeline_call(self, request, **kwargs):
-        rest_request = hasattr(request, "content")
         return_pipeline_response = kwargs.pop("_return_pipeline_response", False)
```

Comment:
Why we explicitly handle "stream" in _pipeline_client.py but not here?

Reply:
talked to @xiangyan99 offline, but this is so we can make the default of `stream` `False` in both sync and async